docs/utilities/idempotency.md

```diff
@@ -148,7 +185,7 @@ You can quickly start by initializing the `DynamoDBPersistenceLayer` class and u
 
 === "Example event"
 
-    ```json
+   ```json
```

Comment:
I think we removed one space here, can we bring it back so we have 4 spaces indentation?
```suggestion
    ```json
```

Reply:
i used a regular tab, is that not ok?

Reply:
You're right, please disregard!